packages/create-sitecore-jss/src/templates/nextjs-styleguide/scripts/scaffold-component/plugins/next-steps.ts

```diff
@@ -0,0 +1,50 @@
+import chalk from 'chalk';
+import { ScaffoldComponentPlugin, ScaffoldComponentPluginConfig } from '..';
+
+/**
+ * Logs next steps. Should be the last plugin to run.
+ */
+class NextStepsPlugin implements ScaffoldComponentPlugin {
+  order = 100;
+
+  exec(config: ScaffoldComponentPluginConfig) {
+    const { componentOutputPath, manifestOutputPath, componentName } = config;
+
+    console.log(
```

Comment:
Might consider adding `nextSteps` prop to `ScaffoldComponentPluginConfig` and console.log those in the `scaffold-component/index.ts`. Similar to what we have in our initializers architecture. This would allow any plugin to add next steps.

Reply:
Good idea, I did that. For the extensibility it's a good approach